src/DebugBar/DataCollector/MemoryCollector.php

```diff
@@ -23,6 +23,18 @@ class MemoryCollector extends DataCollector implements Renderable
 
     protected $peakUsage = 0;
 
+    protected $precision = 0;
```

Comment:
Default value must be 2
https://github.com/maximebf/php-debugbar/blob/376ff32ee50c2c24cebd16ccb5a30c647e9b3c6d/src/DebugBar/DataFormatter/DataFormatter.php#L74